images/chaos-daemon/Dockerfile

```diff
@@ -6,23 +6,20 @@ ARG HTTP_PROXY
 ENV http_proxy $HTTP_PROXY
 ENV https_proxy $HTTPS_PROXY
 
-RUN apt-get update && apt-get install -y tzdata iptables ipset stress-ng iproute2 fuse util-linux procps curl && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && \ 
```

Comment:
We could merge https://github.com/chaos-mesh/chaos-mesh/pull/2340/files first.

Reply:
Merged